src/app.svelte

```diff
@@ -115,6 +115,9 @@
 
         --main-blue: var(--lapis-lazuli);
         --main-black: #585d6e;
+
+        --slider-grey: var(--cultured);
```

Comment:
I don't know if I would name a variable after a specific component since we'll probably want to use that colour in other components/pages too.

Reply:
Actually, why do we even need this variable? Why don't you just use the `--cultured` variable directly?

Reply:
Ahh it's because the value depends on the body class. That makes sense, but I would still rename the variable to something less specific. If the color is specific to that component then I would move the scss styling to the component.

Reply:
I have defined two colors: --ivory: the color of the unfinished progress; --sky-blue: the background of the progressbar thumb in the pressed state